core/src/main/scala/flow/Generator.scala

```diff
@@ -55,6 +55,8 @@ object Generator {
 }
 
 class Generator[+T, +Out](private[flow] val source: () => Future[Source[T, Future[Out]]]) {
+  def futureSource: Future[Source[T, Future[Out]]] = source()
```

Comment:
Doesn't this potentially have a side-effect, since it executes the delayed evaluation of source? I think that is probably unavoidable, but then I think by convention you are supposed to make the signature `futureSource()` (i.e. with the empty parens, to emphasize it has a side effect...? Am I remembering that right?)